{
  "id": "blogeset-blog-eset-com",
  "name": "Blog.Eset",
  "url": "http://blog.eset.com/feed",
  "active": false,
  "created_at": "2025-10-30T16:36:56.721186+00:00",
  "updated_at": "2025-10-30T16:36:56.721194+00:00",
  "metadata": {
    "language": "en",
    "description": "",
    "validation_status": "network_error: HTTPSConnectionPool(host='www.eset.com', port=8443): Max retries exceeded with url: /blog/en/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x1093aae90>: Failed to establish a new connection: [Errno 51] Network is unreachable'))",
    "total_entries": 0,
    "last_checked": "2025-10-30T16:36:56.720882+00:00",
    "import_source": "add_feeds_script"
  }
}